config/unicorn.rb

```diff
@@ -1,3 +1,11 @@
+# Make sure elasticsearch is configured correctly
+require 'rake'
+begin
+  Rake.application['search:validate_index'].invoke
+rescue => e
+  exit(101)
```

Comment:
Also, not sure if there is some way we could make a unit test for this.  If someone ever breaks this, I don't think we'd ever know.